tools/cmake/utility.cmake

```diff
@@ -68,3 +68,20 @@ function(set_macro_definitions)
         endif()
     endforeach()
 endfunction()
+
+macro(set_alias config_name)
```

Comment:
Can you provide documentation for this function on the effects this function has on definition of configs